build_runner/tool/create_test_all.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
```

Comment:
This file probably exists for all packages.... should we stick it in some top level tool dir?